eagle-core/eagle-data-process/eagle-stream-process-api/src/main/java/org/apache/eagle/dataproc/impl/persist/druid/DruidPersistService.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.eagle.dataproc.impl.persist.druid;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigValue;
+import org.apache.eagle.dataproc.impl.aggregate.entity.AggregateEntity;
+import org.apache.eagle.dataproc.impl.persist.IPersistService;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+
+import java.util.*;
+import java.util.concurrent.Future;
+
+/**
+ * TODO : support more general entity input
+ * @since Dec 21, 2015
+ *
+ */
+public class DruidPersistService implements IPersistService<AggregateEntity> {
```

Comment:
How do you think to rename the `DruidPersistService` to `KafkaPersistService` here as it's for kafka only in fact. We may have another `DruidPersistService` using https://github.com/druid-io/tranquility later.